core/frontend/src/IModelConnection.ts

```diff
@@ -1020,3 +1029,26 @@ export namespace IModelConnection { // eslint-disable-line no-redeclare
     }
   }
 }
+
+export class ProjectAltitudeProvider {

```

Comment:
Missing release tag; supposed to be `@public`? If not, need release tag on IModelConnection.altitudeProvider.

Reply:
Fixed.